torch/jit/_script.py

```diff
@@ -898,6 +898,10 @@ def forward(self, input):
             obj, torch.jit._recursive.infer_methods_to_compile
         )
 
+    # for partial objects we need to extract the function for scripting
+    if isinstance(obj, functools.partial):
```

Comment:
Ditto:

IMHO, user's expectation would be scripting partial function, not the underlying original function. Let me know what you think.